src/rules/no-useless-path-segments.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileOverview Ensures that there are no useless path segments
+ * @author Thomas Grainger
+ */
+
+import path from 'path'
+import sumBy from 'lodash.sumby'
+import resolve from 'eslint-module-utils/resolve'
+import moduleVisitor from 'eslint-module-utils/moduleVisitor'
+
+/**
+ * convert a potentialy relative path from node utils into a true
```

Comment:
potentially

Reply:
Weird, I thought I had a test case for that. That's what I added https://github.com/benmosher/eslint-plugin-import/pull/912/commits/9f3c3a7d91aa3b3428bbe392f95be2239edcdf57#diff-8b276518a9cbd15a6895feee0de942b1 for. I'll add it back in and fix the typo.